### PR TITLE
Set the USER env for e2e jobs

### PR DESCRIPTION
**Pull request by flaper87**
Sadly, the gophercloud/utils lib uses user.CurrentUser to load get the
user home dir. This fails with a panic error when used in environments
where such functionality is not available.

Let's set the `USER` env var for now until I fix this issue in
gophercloud.

### CODE REVIEW

**Comment by wking** [comment]
More on this issue in a different context [here][1] and in #1185.

[1]: https://github.com/openshift/release/pull/1178#issuecomment-415213896

**Comment by flaper87** [comment]
@wking Thanks! I added HOME although I don't think I need it for this specific fix.

File: ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml [hunk 1]
```diff
@@ -256,6 +256,10 @@ objects:
         value: openstack-cloud
       - name: OS_CLIENT_CONFIG_FILE
         value: /etc/openshift-installer/clouds.yaml
+      - name: USER
+        value: test
+      - name: HOME
+        value: /home/test
```
Comment (wking) [inline ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml]: Do we have permission to create this?  In #1185, I'd just set `HOME` to `/tmp`.

Comment (flaper87) [reply ci-operator/templates/openshift/installer/cluster-launch-installer-e2e.yaml]: mmh, good point. I was gonna set it to tmp and then changed my mind!

**Comment by flaper87** [comment]
@wking done! Thanks a bunch for the review :smiley:

**Pull request merged**